module/zfs/zfs_vfsops.c

```diff
@@ -453,24 +458,40 @@ zfs_userspace_many(zfs_sb_t *zsb, zfs_userquota_prop_t type,
 	zap_attribute_t za;
 	zfs_useracct_t *buf = vbuf;
 	uint64_t obj;
+	int offset = 0;
 
 	if (!dmu_objset_userspace_present(zsb->z_os))
```

Comment:
If we can assume the dependency between space and object accounting, then it should be okay as-is.